test/Feature/Holistic/introspection/schemaTypes/__DirectiveLocation/response.json

```diff
@@ -8,92 +8,92 @@
       "interfaces": null,
       "enumValues": [
         {
-          "name": "QUERY",
+          "name": "S__DirectiveLocationQUERY",
```

Comment:
this, should not happen.

we must define custom GQL instances(Introspect, Encode, Decode) for Enums.

Reply:
we can prefix enums for Server in another merge Request (just remove Prefixing from Document), you don't have to do all of this. 


but if you wish to implement this too. i can show you where to start.